src/components/common/ColonyActions/ActionsListItem/config/helpers.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
```

Comment:
I can see this file being merged with `config.ts` in the event the data wrangling requirements are simplified (e.g. no async) and we don't have much to handle in the component. But for now I've separated as it makes it easier to see what's going on in `config`. 

This file is basically just responsible for ensuring that the items object has key value pairs that match what we're expecting in our message descriptors.